src/Platforms/MySQL/Comparator.php

```diff
@@ -30,11 +31,12 @@ public function __construct(
         parent::__construct($platform);
     }
 
-    public function compareTables(Table $oldTable, Table $newTable): TableDiff
+    public function compareTables(Table $oldTable, Table $newTable, ?ComparatorConfig $config = null): TableDiff
```

Comment:
As I mentioned in the issue, Comparator is indeed an extensible class, so changing the public signature is a breaking change, though likely a very low impact one

Reply:
So, should I add a `setConfig()` method to the `Comparator`?

Reply:
Actually from some testing it would not be a breaking change, php allows signatures to differ as long as the parameter is optional (which this one is)

```php
class A {
	
	function abc($a) {
		
	}
}

class B extends A {
	
	function abc($a, $b = null) {
		
	}
}

class C extends A {
	
	function abc($a) {
		
	}
}
```

Reply:
I think it would be best yes

Reply:
Done in d0c8602b4e2ce9fcf5baba7e72d7781e7c4340b6